manifests/params.pp

```diff
@@ -34,9 +37,12 @@
       $ldap_auth_plugin_location = undef # no ldap plugin on redhat/centos
     }
     'Debian': { # Debian/Ubuntu
-      $group = 'nogroup'
-      $link_openssl_cnf = true
-      $pam_module_path = '/usr/lib/openvpn/openvpn-auth-pam.so'
+      $etc_directory     = '/etc'
+      $root_group        = 'root'
+      $group             = 'nogroup'
+      $link_openssl_cnf  = true
+      $pam_module_path   = '/usr/lib/openvpn/openvpn-auth-pam.so'
+      $load_main_service = true
```

Comment:
What is this variable used for? I cannot find any reference to it? If it's needed, we have to add it to `Archlinux` and `RedHat` as well, don't we?